src/screen/AddAccountScreen/AddAccountScreen.tsx

```diff
@@ -80,14 +80,14 @@ export function AddAccountScreen({navigation}: {navigation: NavigationProp<AppSt
   return (
     <Modalize
       ref={ref}
-      withOverlay={true}
-      threshold={250}
-      scrollViewProps={{showsVerticalScrollIndicator: false}}
       adjustToContentHeight
-      onClose={navigation.goBack}
-      handlePosition="outside"
+      onClose={() => {
+        // timeout fixes a warning about animation event handling
+        // not sure what is the problem
+        setTimeout(navigation.goBack, 0);
```

Comment:
Is the issue only happening in this screen? We are doing the same in some other places

Reply:
yes, I saw it only here. for example on `MyIdentity` screen and `DevScreen` it was fine.